views/index.jade

```diff
@@ -33,31 +33,20 @@ block scenarios
         -
           var scenario = scenario_session.scenario,
             session = scenario_session.session;
-        if session
-          if session.instance || user.instructor
-            tr.session
-              td.col-sm-5= session.state.scenarioTitle
-                br
-                span.small= instructorStudentsLabel( session )
-              td.col-sm-1= session.state.classroom.company
-              td.col-sm-1= session.state.classroom.platoon
-              td.col-sm-1= session.state.classroom.unit
-              td.col-sm-2 &nbsp;
-              td.col-sm-1: a.btn.btn-sm.join(href=session.instance||session.document.uri target="_blank")= session.instance ? "Join" : "Start"
-              th.col-sm-1 &nbsp;
-        else if user.instructor
-          tr.scenario
-            form
-              td.col-sm-5= scenario.state.scenarioTitle
-              td.col-sm-1.company: input.form-control.input-sm(name="company" type="text" maxlength="8")
-              td.col-sm-1.platoon: input.form-control.input-sm(name="platoon" type="number" min="1" max="9" step="1")
-              td.col-sm-1.unit: input.form-control.input-sm(name="unit" type="number" min="1" max="9" step="1")
-              td.col-sm-2.name: input(name="name" type="hidden" value=scenario.state.scenarioName)
-              td.col-sm-1
-                a.btn.btn-sm.edit(href=scenario.instance||scenario.document.uri target="_blank") Edit
-                button.btn.btn-sm.create.hidden(type="submit" disabled) Create
-              td.col-sm-1
-                a.btn.btn-sm.export(href="/export-scenarios?scenarioName=" + scenario.state.scenarioName) Export
+        unless session
+          if user.instructor
```

Comment:
You can remove this `user.instructor` test and the one above now since this block is only shown for instructors. This is consistent with the _sessions_ (now _review_) block, which also doesn't check for the instructor.